code/_onclick/telekinesis.dm

```diff
@@ -140,10 +140,6 @@
 		to_chat(user, "<span class='notice'>This object is too heavy to move with something buckled to it!</span>")
 		return
 
-	if(length(focus.client_mobs_in_contents))
-		to_chat(user, "<span class='notice'>This object is too heavy to move with something inside of it!</span>")
-		return
```

Comment:
I know you are removing mobs in but this logic should be somewhat preserved.
Perhaps loop for mob in contents as the last if in this chain?

Reply:
fair enough